cupyx/scipy/linalg/decomp_lu.py

```diff
@@ -0,0 +1,212 @@
+from warnings import warn
+
+import numpy
+
+import cupy
+from cupy import cuda
+from cupy.cuda import cublas
+from cupy.cuda import device
+from cupy.linalg import util
+
+if cuda.cusolver_enabled:
+    from cupy.cuda import cusolver
+
+
+def lu_factor(a, overwrite_a=False, check_finite=True):
+    """LU decomposition.
+
+    Decompose a given two-dimensional square matrix into ``P * L * U``,
+    where ``P`` is a permutation matrix,  ``L`` lower-triangular with
+    unit diagonal elements, and ``U`` upper-triangular matrix.
+    Note that in the current implementation ``a`` must be
+    a real matrix, and only float32 and float64 are supported.
```

Comment:
```suggestion
    a real matrix, and only :class:`numpy.float32` and :class:`numpy.float64` are supported.
```
(The above suggestion may exceed 80 characters.)

Reply:
Thanks for the suggestion.
I modified as you suggested.